requirements.txt

```diff
@@ -0,0 +1,13 @@
+click==6.6
+Flask==0.11.1
+Flask-Login==0.4.0
+Flask-SQLAlchemy==2.1
+Flask-WTF==0.13.1
+itsdangerous==0.24
+Jinja2==2.8
+MarkupSafe==0.23
+requests==2.12.4
+scrypt==0.8.0
+SQLAlchemy==1.1.4
+Werkzeug==0.11.11
+WTForms==2.1
```

Comment:
- please don't use frozen dependency versions, its meant to work with the latest. If not, its a bug.
- please only use direct first-level dependencies and purge the transitive dependencies like click, MarkupSafe etc...